packages/fmg-simple-adjudicator/contracts/SimpleAdjudicator.sol

```diff
@@ -190,10 +234,19 @@ contract SimpleAdjudicator {
       public
       onlyWhenGameTerminated
     {
-        require(
-            _channelId == fundedChannelId,
-            "Can only withdraw from the funded channel id"
-        );
+        return _withdraw(participant, destination, _channelId,v,r,s);
+    }
+
+    function _withdraw(
+        address participant,
+        address destination,
+        bytes32 _channelId, // not needed for the simple adjudicator, which only supports one channel
+        uint8 v,
+        bytes32 r,
+        bytes32 s
+    )
+      internal
```

Comment:
Should the `onlyWhenGameTerminated` modifier be here instead? Seems like we'd never want to call `_withdraw` when it isn't.